apps/molgenis-components/src/components/forms/InputSelect.vue

```diff
@@ -12,10 +12,10 @@
     <select
       v-else-if="!readonly"
       :id="id"
-      :modelValue="modelValue"
+      :modelValue="defaultModelValue(modelValue, required, options)"
       :readonly="readonly"
       class="form-control"
-      @change="$emit('update:modelValue', $event.target.value)"
+      @change="updateModelValue($event.target.value)"
     >
       <option
```

Comment:
I think a better solution would be to add an empty option?

Reply:
Indeed. The issue here is that the behaviour between "never clicked on any other option" vs "clicked a different option and then selected the empty option again" was different.